Architecture/TodoApp/android/core/src/test/kotlin/com/novoda/todoapp/tasks/service/PersistedTasksServiceTest.kt

```diff
@@ -1081,15 +1200,22 @@ class PersistedTasksServiceTest {
         return Tasks.from(ImmutableList.copyOf(listOf<SyncedData<Task>>(
                 SyncedData.from(Task.builder().id(Id.from("24")).title("Bar").isCompleted(true).build(), syncState, time),
                 SyncedData.from(Task.builder().id(Id.from("42")).title("Foo").build(), SyncState.IN_SYNC, TEST_TIME),
-                SyncedData.from(Task.builder().id(Id.from("12")).title("Whizz").build(), SyncState.IN_SYNC, TEST_TIME),
-                SyncedData.from(Task.builder().id(Id.from("424")).title("New").isCompleted(true).build(), SyncState.IN_SYNC, TEST_TIME)
+                SyncedData.from(Task.builder().id(Id.from("424")).title("New").isCompleted(true).build(), SyncState.IN_SYNC, TEST_TIME),
+                SyncedData.from(Task.builder().id(Id.from("425")).title("Whizz").build(), SyncState.IN_SYNC, TEST_TIME)
         )))
     }
 
     private fun asSyncedTasks(remoteTasks: List<Task>) = Tasks.asSynced(remoteTasks, TEST_TIME)
 
     private fun noEmptyTasks() = NoEmptyTasksPredicate()
 
+
+    private fun givenPresenterWithTestScheduler(): TestScheduler {
```

Comment:
Reading this gives me the idea that it returns a presenter. If you still want to return a `TestScheduler`, instead of setting it as an object property, you can refactor it to `givenPresenterAndReturnTestScheduler` or smt like that, maybe?